File: Library/JS/Global/Global.js





/**
 * 输出信息到控制台
这是为了方便输出信息而设计的便捷函数
 * @param {any} data1? 
 * @param {any} data2? 
 * @param {} data3? 
 * @returns {null} 
 */
 function log(data1,data2,data3)

/**
 * 输出带颜色文本
 * @param {string} color 此行输出的颜色
 * @param {any} data1? 
 * @param {any} data2? 
 * @returns {null} 
 */
 function colorLog(color,data1,data2)

/**
 * 推迟一段时间执行函数
 * @param {(:)any} func 待执行的函数
 * @param {number} msec 推迟执行的时间（毫秒）
 * @returns {number} 此任务ID
 */
 function setTimeout(func,msec)

/**
 * 推迟一段时间执行代码段（eval）
 * @param {string} code 待执行的代码段
 * @param {number} msec 推迟执行的时间（毫秒）
 * @returns {number} 此任务ID
 */
 function setTimeout(code,msec)

/**
 * 设置周期执行函数
 * @param {()any} func 待执行的函数
 * @param {number} msec 执行间隔周期（毫秒）
 * @returns {number} 此任务ID
 */
 function setInterval(func,msec)

/**
 * 设置周期执行代码段（eval）
 * @param {string} code 待执行的代码段
 * @param {number} msec 执行间隔周期（毫秒）
 * @returns {number} 此任务ID
 */
 function setInterval(code,msec)

/**
 * 取消延时 / 周期执行项
 * @param {number} timerid 由前几个函数返回的任务ID
 * @returns {boolean|null} 是否取消成功
 */
 function clearInterval(timerid)

File: Library/Library/Lua/Global/Global.lua


---输出信息到控制台
---这是为了方便输出信息而设计的便捷函数
---@param data1? any 
---@param data2? any 
---@param data3?  
---@return nil 
function log(data1,data2,data3)
end

---输出带颜色文本
---@param color string 此行输出的颜色
---@param data1? any 
---@param data2? any 
---@return nil 
function colorLog(color,data1,data2)
end

---推迟一段时间执行函数
---@param func fun(:):any 待执行的函数
---@param msec number 推迟执行的时间（毫秒）
---@return number 此任务ID
function setTimeout(func,msec)
end

---推迟一段时间执行代码段（eval）
---@param code string 待执行的代码段
---@param msec number 推迟执行的时间（毫秒）
---@return number 此任务ID
function setTimeout(code,msec)
end

---设置周期执行函数
---@param func fun():any 待执行的函数
---@param msec number 执行间隔周期（毫秒）
---@return number 此任务ID
function setInterval(func,msec)
end

---设置周期执行代码段（eval）
---@param code string 待执行的代码段
---@param msec number 执行间隔周期（毫秒）
---@return number 此任务ID
function setInterval(code,msec)
end

---取消延时 / 周期执行项
---@param timerid number 由前几个函数返回的任务ID
---@return boolean|nil 是否取消成功
function clearInterval(timerid)
end